ports/virtualgl/portfile.cmake

```
@@ -0,0 +1,39 @@
+vcpkg_from_github(
+    OUT_SOURCE_PATH SOURCE_PATH
+    REPO VirtualGL/virtualgl
+    REF "${VERSION}"
+    SHA512 4668ebfbfe663d3cf66468b120ecf46da161cbce3b793dbeaea9ace50d19d8d3edcdc84ef31d87b27903c1c8a6648ae0419c796960928f1421a90171a5db7d43
+    HEAD_REF main
+    PATCHES
+        fix-dependencies.patch
+)
+
+vcpkg_check_features(OUT_FEATURE_OPTIONS FEATURE_OPTIONS
+    FEATURES
+        "opencl"    VGL_FAKEOPENCL
+        "xcb"       VGL_FAKEXCB
+        "xv"        VGL_USEXV
+)
+
+if(VCPKG_LIBRARY_LINKAGE STREQUAL "static")
+    set(TJPEG_LIBRARY "${_VCPKG_INSTALLED_DIR}/${TARGET_TRIPLET}/lib/libturbojpeg.a")
+elseif(VCPKG_LIBRARY_LINKAGE STREQUAL "dynamic")
+    set(TJPEG_LIBRARY "${_VCPKG_INSTALLED_DIR}/${TARGET_TRIPLET}/lib/libturbojpeg.so")
+endif()
+
+vcpkg_cmake_configure(
+    SOURCE_PATH "${SOURCE_PATH}"
+    OPTIONS
+        -DTJPEG_INCLUDE_DIR="${_VCPKG_INSTALLED_DIR}/${TARGET_TRIPLET}/include"
+        -DTJPEG_LIBRARY="${TJPEG_LIBRARY}"
```

Quote for CMake, not for some particular shell:
```suggestion
        "-DTJPEG_INCLUDE_DIR=${_VCPKG_INSTALLED_DIR}/${TARGET_TRIPLET}/include"
        "-DTJPEG_LIBRARY=${TJPEG_LIBRARY}"
```
(Well, the second one should go away anyways, and the first one could be handled in the same spot, for locality.)

---

Pushed an update